PureBasicIDE/FindWindow.pb

```diff
@@ -32,10 +33,17 @@ Procedure OpenFindWindow()
         SetGadgetState(#GADGET_Find_SelectionOnly,FindSelectionOnly)
         SetGadgetState(#GADGET_Find_AutoWrap,     FindAutoWrap)
         
-        SetGadgetState(#GADGET_Find_DoReplace,  0) ; doreplace should always be disabled when opening this window.
-        DisableGadget(#GADGET_Find_ReplaceWord, 1)
-        DisableGadget(#GADGET_Find_Replace, 1)
-        DisableGadget( #GADGET_Find_Replaceall, 1)
+        If EventMenu() = #MENU_Replace
```

Comment:
Having the EventMenu() check here means that the whole `OpenFindWindow()` procedure can only be called in reaction to a menu event. In the interest of future flexibility this should be changed. I suggest to add a parameter for this to `OpenFindWindow()` and call it with the proper parameter depending on the menu event, just like it is done with `OpenProjectOptions()` for example.

Reply:
I hesitated indeed between adding a parameter (Replace = #False) or MenuEvent(), as it is called only from UserInterface.pb
OK, I'll change it.